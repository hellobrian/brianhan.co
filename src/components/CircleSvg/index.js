import styled from 'styled-components';

const CircleSvg = styled.svg`
  background-image: var(--gradient--salmon);
  width: 7rem;
  height: 7rem;
  border-radius: 50%;
`;

export default CircleSvg;
