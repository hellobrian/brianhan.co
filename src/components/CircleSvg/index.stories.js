import React from 'react'
import { storiesOf } from '@storybook/react'
import { withKnobs } from '@storybook/addon-knobs'
import CircleSvg from './index'

const stories = storiesOf('CircleSvg', module)
stories.addDecorator(withKnobs)
stories.add('default', () => <CircleSvg />)
