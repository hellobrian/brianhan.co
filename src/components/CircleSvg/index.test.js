import React from 'react';
import { render } from 'react-testing-library';
import CircleSvg from './index';

describe('CircleSvg', () => {
  it('renders correctly', () => {
    const { container } = render(<CircleSvg />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
