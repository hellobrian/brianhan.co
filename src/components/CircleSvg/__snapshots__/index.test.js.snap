// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircleSvg renders correctly 1`] = `
.c0 {
  background-image: var(--gradient--salmon);
  width: 7rem;
  height: 7rem;
  position: fixed;
  border-radius: 50%;
  top: -1rem;
  left: -1rem;
}

<svg
  class="c0"
/>
`;
